plugins/faustwp/readme.txt

```diff
@@ -1,18 +1,18 @@
-=== Faust ===
+=== Faust.js ===
 Contributors: antpb, apmatthe, blakewpe, chriswiegman, claygriffiths, jasonkonen, joefusco, markkelnar, matthewguywright, mindctrl, modernnerd, rfmeier, TeresaGobble, thdespou, wpengine
-Tags: faustjs, faust, headless, decoupled
+Tags: faustjs, faust, headless, decoupled, composable-architecture
 Requires at least: 5.7
 Tested up to: 6.2
 Stable tag: 0.8.5
 Requires PHP: 7.2
 License: GPLv2 or later
 License URI: https://www.gnu.org/licenses/gpl-2.0.html
 
-Faust transforms your traditional WordPress installation into a flexible headless CMS.
+Faust.js™ transforms your traditional WordPress installation into a flexible headless CMS.

```

Comment:
This will be the first reference of the trademarked name for https://wordpress.org/plugins/faustwp/, meaning that any reference to the trademark that comes after this does not require ™